lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -356,7 +360,7 @@ export class UserAgentApplication {
     this.userLoginInProgress = true;
 
     // TODO: Make this more readable - is authorityInstance changed, what is happening with the return for AuthorityKey?
-    this.authorityInstance.ResolveEndpointsAsync().then(() => {
+    this.authorityInstance.resolveEndpointsAsync().then(() => {
```

Comment:
there are a lot of changes here unrelated to test

Reply:
:). Me and Prithvi merged the first cut, this is something he changed the case of the function to be consistent.